src/python/pants/core/util_rules/environments.py

```diff
@@ -171,9 +171,18 @@ class DockerImageField(StringField):
     )
 
 
+class DockerPlatformField(StringField):
+    alias = "platform"
+    required = True
+    valid_choices = Platform
+    help = (
+        f"The intended platform for the Docker image, one of: {[plat.value for plat in Platform]}"
+    )
```

Comment:
This renders as:

```
platform
    type: 'linux_arm64' | 'linux_x86_64' | 'macos_arm64' | 'macos_x86_64'
    required

    The intended platform for the Docker image, one of: ['linux_arm64',
    'linux_x86_64', 'macos_arm64', 'macos_x86_64']
```

So it's not necessary to repeat the platform choices. But I was thinking maybe it's a good idea because these platform values are not the same values as Docker's `--platform` arg?

--

I'm not sure if `help` should mention anything more here